src/renderer/app/client.ts

```diff
@@ -23,7 +24,10 @@ const authLink = new ApolloLink((operation, forward) => {
     }
   })
   // @ts-ignore
-  return forward(operation)
+  return forward(operation).map((response) => {
+    response.data = uncrunch(response.data)
```

Comment:
Maybe check if data is not undefined?

Reply:
The same in guildspeak/guildspeak-backend#17

Reply:
look at `   if (!operation) throw new Error('operation error')`

Reply:
So... are you sure data won't be undefined?

Reply:
also Prisma automatically throw error if data is not undefined

Reply:
Alright, approving then.